bundles/com.espressif.idf.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -54,4 +53,4 @@ command.label.10 = ESP-IDF: Python Clean
 command.tooltip.1 = Advised to run this after switching between IDF and Python versions.Generated python byte code can be deleted from the IDF directory
 command.name.12 = ESP-IDF: Python Clean
 command.name.11 = Delete
-
+command.name.14 = Update ESP-IDF master
```

Comment:
Can you please name the property to reflect what it is referencing to meaningfully?

Reply:
Hi Ali, changed to command.name.UpdateEspIdfMaster. Thanks